ts/components/LandingCard.tsx

```diff
@@ -27,17 +27,20 @@ const style = StyleSheet.create({
     width: screenWidth / 2,
     height: screenWidth / 2,
     resizeMode: "contain"
+  },
+  text: {
+    fontSize: 20
   }
 });
 
 export const LandingCardComponent: React.SFC<Props> = card => (
-  <View style={style.card}>
-    <Image source={card.image} style={style.image} />
+  <View style={styles.card}>
+    <Image source={card.image} style={styles.image} />
     <View spacer={true} />
     <Grid>
       <Col size={1} />
       <Col size={7}>
-        <Text bold={true} alignCenter={true}>
+        <Text bold={true} alignCenter={true} style={styles.text}>
           {" "}
           {card.title}{" "}
```

Comment:
`{" "}` items seem to be unused. It does not seem to be the case but, if a vertical spacer is needed, you could add a `<View spacer={true}/>`

Reply:
They were already there but in fact it seems to me that they are of no use and the spacer is not needed either. I remove its